binary_search_tree/binary_search_tree.py

```diff
@@ -5,10 +5,44 @@ def __init__(self, value):
     self.right = None
 
   def insert(self, value):
-    pass
-
+    if self.value: 
+      if value < self.value:
+        if self.left is None:
+          self.left = BinarySearchTree(value)
+        else:
+          self.left.insert(value)
+      if value >= self.value:
+        if self.right is None:
+          self.right = BinarySearchTree(value)
+        else:
+          self.right.insert(value)
+    else:
+      self.value = value
+  
   def contains(self, target):
-    pass
+    if target == self.value:
+      return True
+    elif target > self.value:
+      if self.right is None:
+        return False
+      elif target == self.right.value:
+        return True
+      else:
+        self.right.contains(target)
+    elif target < self.value:
+      if self.left is None:
+        return False
+      elif target == self.left.value:
+        return True
+      else:
+        self.left.contains(target)
+    else:
+      return False
+
```

Comment:
Hey Katia, your binary search code implementation is neat, my feedback will be to always delete 'pass' when you start working on all your functions.